evm/src/libraries/TransceiverStructs.sol

```diff
@@ -18,8 +18,20 @@ library TransceiverStructs {
     /// @dev Selector 0x56d2569d.
     /// @param prefix The prefix that was found in the encoded message.
     error IncorrectPrefix(bytes4 prefix);
+
+    /// @notice Error thrown when the transceiver instructions aren't
+    ///         encoded with strictly increasing indices
+    /// @dev Selector 0x71f23ef2.
     error UnorderedInstructions();
```

Comment:
might be good to include the `lastIndex` and the `instructionIndex` in the error